samples/storage_csm_powerstore_v250.yaml

```diff
@@ -27,6 +27,7 @@ spec:
       # Allowed values: ReadWriteOnceWithFSType, File , None
       # Default value: ReadWriteOnceWithFSType
       fSGroupPolicy: "ReadWriteOnceWithFSType"
+      storageCapacity: true
```

Comment:
Please describe the of variable storageCapacity ref: fSGroupPolicy

Reply:
Done